doc/config.md

```diff
@@ -39,3 +39,16 @@ Lastly, `lein-monolith` supports inheriting unprocessed values, via
 inheriting paths, as Leiningen absolutizes paths upon processing a project map.
 By using raw inheritance, you can safely inherit paths, e.g. `:test-paths` or
 `:source-paths`.
+
+## Dependency Sets
+
+The `:monolith/dependency-set` key can be used to opt projects into a specific
+set of managed dependencies instead of using a single list of managed dependencies
+in the metaproject.
+
+Dependency sets are defined in the metaproject with the `:dependency-sets` key.
```

Comment:
It's not clear from reading this what the structure of this should be. It'd be good to expand this and maybe provide a simple example.